scripts/create-certs.sh

```diff
@@ -0,0 +1,43 @@
+# Create a Root Certificate (self-signed):
+
+# Generate a private key for the Root CA
+openssl ecparam -name prime256v1 -genkey -noout -out root_key.pem
+
+# Create a self-signed Root certificate
+openssl req -new -x509 -key root_key.pem -out root_cert.pem -days 3650
+
+# Create an Intermediate CA Certificate (signed by the Root, and itself a CA):
+
+# Generate a private key for the Intermediate CA
+openssl ecparam -name prime256v1 -genkey -noout -out intermediate_key.pem
+
+# Create a certificate signing request for the Intermediate CA
+openssl req -new -key intermediate_key.pem -out intermediate_csr.pem
+
+# Create a configuration file (intermediate_ca.cnf) to specify the basicConstraints extension
+echo "basicConstraints=CA:TRUE" > intermediate.cnf
+
+# Sign the Intermediate CA certificate with the Root, specifying the basicConstraints extension
+openssl x509 -req -in intermediate_csr.pem -CA root_cert.pem -CAkey root_key.pem -out intermediate_cert.pem -days 1825 -extfile intermediate.cnf
```

Comment:
Break this line up some, and while you're at it put that configuration inline:
```suggestion
# Sign the Intermediate CA certificate with the Root, specifying the basicConstraints extension
openssl x509 -req \
    -in intermediate_csr.pem \
    -CA root_cert.pem -CAkey root_key.pem \
    -out intermediate_cert.pem \
    -days 1825 \
    -extfile <(echo 'basicConstrants=CA:TRUE')
```
(also note use of single quotes for echoing the constraints extension, since there's no interpolation to be done)